tests/quantization/test_saturation_export.py

```diff
@@ -0,0 +1,213 @@
+from nncf import NNCFConfig
+from nncf.layers import NNCFConv2d
+from tests.helpers import create_conv, create_compressed_model_and_algo_for_test
+from nncf.quantization.layers import QuantizerConfig, QuantizationMode, SymmetricQuantizer, AsymmetricQuantizer
+
+
+def test_is_correct_saturation_fix_levels():
+    q_config = QuantizerConfig(
+        bits=8,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=False,
+        is_weights=True,
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 128)
+    assert (quantizer.level_low == -64)
+    assert (quantizer.level_high == 63)
+
+    q_config = QuantizerConfig(
+        bits=8,
+        mode=QuantizationMode.ASYMMETRIC,
+        signedness_to_force=True,
+        per_channel=False,
+        is_weights=True,
+        is_saturation_fix=True)
+
+    quantizer = AsymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 128)
+    assert (quantizer.level_low == 0)
+    assert (quantizer.level_high == 127)
+
+    q_config = QuantizerConfig(
+        bits=7,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=True,
+        is_weights=True,
+        input_shape=[3, 32, 32],
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 64)
+    assert (quantizer.level_low == -32)
+    assert (quantizer.level_high == 31)
+
+    q_config = QuantizerConfig(
+        bits=4,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=False,
+        is_weights=True,
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 8)
+    assert (quantizer.level_low == -4)
+    assert (quantizer.level_high == 3)
+
+    q_config = QuantizerConfig(
+        bits=8,
+        mode=QuantizationMode.SYMMETRIC,
+        signedness_to_force=True,
+        per_channel=False,
+        is_weights=False,
+        is_saturation_fix=True)
+
+    quantizer = SymmetricQuantizer(q_config)
+
+    assert (quantizer.is_saturation_fix == True)
+    assert (quantizer.levels == 128)
+    assert (quantizer.level_low == -64)
+    assert (quantizer.level_high == 63)
+
+
+def get_config_for_export_mode(should_be_onnx_standard: bool) -> NNCFConfig:
+    nncf_config = NNCFConfig()
+    nncf_config.update({
+        "input_info": {
+            "sample_size": [1, 1, 20, 20]
+        },
+        "compression": {
+            "algorithm": "quantization",
+            "export_to_onnx_standard_ops": should_be_onnx_standard
+        },
+        "target_device": "CPU"
+    })
+    return nncf_config
+
+
+class TwoConvTestModel(nn.Module):
```

Comment:
Is this a copy of an existing test model in other files?

Reply:
Now use existing model